src/rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date)
+    @date = date
+    @person = person
+    @book = book
```

Comment:
- [ ] Please ensure that the `initialize` method accepts the `person` and `book` in the correct order in order to successfully create a rental 👍